dev-tools/src/main/resources/forbidden/all-signatures.txt

```diff
@@ -128,6 +128,9 @@ com.google.common.collect.FluentIterable
 com.google.common.io.Files
 com.google.common.primitives.Ints
 
+#com.google.common.collect.ImmutableSet
+com.google.common.collect.ImmutableMap#entrySet()
+
```

Comment:
Would it be okay to drop out the commented out `om.google.common.collect.ImmutableSet` and move the `com.google.common.collect.ImmutableMap#entrySet()` so that it's together with the rest of the Guava bans and therefore gets the same message as the rest of them?